pkg/matchers.go

```diff
@@ -18,15 +18,21 @@ func IsInstalled(pkgs ...Pkg) error {
 			return fmt.Errorf("Unable to look up %s: no package found", p)
 		}
 
-		if p.Version() == "" {
+		if p.Version() == nil {
 			continue
 		}
 
+	CheckPkgs:
 		for _, foundPkg := range foundPkgs {
-			if foundPkg.Version() == p.Version() {
-				continue
+			for _, foundPkgVersion := range foundPkg.Version() {
+				for _, pkgVersion := range p.Version() {
```

Comment:
I need to think this one through. Even when I was writing these codezzzz it felt unnatural to me :-/ The possibility of checking multiple versions is real because of how `gem` works. Whether we should provide that possibility to a use is a whole other question though :-)